src/org/traccar/api/resource/NotificationResource.java

```diff
@@ -57,4 +60,11 @@ public Response update(Notification entity) throws SQLException {
         return Response.ok(entity).build();
     }
 
+    @Path("testmail")
+    @GET
+    public Response testMail() throws MessagingException {
+        NotificationMail.sendMailSync(getUserId(), new Event("unknown", 0), null);
```

Comment:
Frankly, it does not really mater. Just should disagree with existent types.
The email will come with "Unknown type" text.